docker/Dockerfile

```diff
@@ -39,8 +39,9 @@ RUN adduser appuser && \
     make build 
 
 # https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
-FROM registry.access.redhat.com/ubi8-minimal:8.3-230
+FROM registry.access.redhat.com/ubi8-minimal:8.3-298
 USER root
+RUN microdnf -y update && microdnf clean all && rm -rf /var/cache/yum && echo "Installed Packages" && rpm -qa | sort -V && echo "End Of Installed Packages"
```

Comment:
no || true here for microdnf update ?

Reply:
same as in other repos... if we can't update, we ought to break.

Reply:
can't rememeber why I had the ``||true`` in https://github.com/che-incubator/configbump/pull/17/files#diff-41dbb919b29e065cd27734d8305262c1f450927af5b6629c0eb4261357c95a67R24 ... must have been some odd missing dependency thing.